.NET/Microsoft.Recognizers.Definitions/Chinese/NumbersWithUnitDefinitions.cs

```diff
@@ -244,6 +244,7 @@ public static class NumbersWithUnitDefinitions
 			{ "Finnish markka", "芬兰马克" },
 			{ "Penni", "盆尼" }
 		};
+		public const string CompoundUnitRegex = @"(?<spacer>又|再)";

```

Comment:
This is the connector between main unit and factional unit, right?
I'd prefer to rename to CompoundUnitConnectorRegex